examples/stress.rs

```diff
@@ -126,7 +127,8 @@ async fn main() -> Result<()> {
     let mut churn_events = schedule.events();
 
     let probe_interval = Duration::from_secs_f64(1.0 / opts.probe_frequency);
-    let mut probes = time::interval(probe_interval);
+    let probes = time::interval(probe_interval);
+    let mut probe_stream = IntervalStream::new(probes);
```

Comment:
We don't need this. Just use [`tick`](https://docs.rs/tokio/1.2.0/tokio/time/struct.Interval.html#method.tick) instead of `next`. We probably don't need `tokio_stream` at all.